external/include/mbedtls/config.h

```diff
@@ -2520,7 +2520,7 @@
 
 /* MPI / BIGNUM options */
 //#define MBEDTLS_MPI_WINDOW_SIZE            6 /**< Maximum windows size used. */
-#define MBEDTLS_MPI_MAX_SIZE            512	/**< Maximum number of bytes for usable MPIs. */
+#define MBEDTLS_MPI_MAX_SIZE            CONFIG_TLS_MPI_MAX_SIZE	/**< Maximum number of bytes for usable MPIs. */
```

Comment:
You didn't modify defconfigs for this so that you should use below:
```
#ifdef CONFIG_XX
#define YY CONFIG_XX
#else
#define YY 512
#endif

Reply:
modify